experiments/ClimaEarth/test/component_model_tests/climaland_tests.jl

```diff
@@ -46,3 +52,65 @@ FT = Float32
     driver_names = propertynames(land_sim.integrator.p.drivers)
     @test driver_names == (:P_liq, :P_snow, :c_co2, :T, :P, :q, :SW_d, :LW_d, :cosθs, :frac_diff, :soc)
 end
+
+@testset "ClimaLandSimulation flux calculations" begin
+    dt = Float64(120)
+    tspan = (Float64(0), 3.0dt)
+    start_date = Dates.DateTime(2008)
+    output_dir = pwd()
+
+    # Construct atmos and land simulation objects
+    atmos_config_file = joinpath(exp_dir, "test", "component_model_tests", "climaatmos_coarse_short.yml")
+    atmos_config = CA.AtmosConfig(atmos_config_file; job_id = "atmos_land_flux_test")
+    atmos_sim = ClimaAtmosSimulation(atmos_config)
+
+    boundary_space = ClimaCore.Spaces.horizontal_space(atmos_sim.domain.face_space)
+    area_fraction = ClimaCore.Fields.ones(boundary_space)
+    land_sim = ClimaLandSimulation(FT, dt, tspan, start_date, output_dir, boundary_space, area_fraction)
+    model_sims = (; land_sim = land_sim, atmos_sim = atmos_sim)
+
+    # Construct a coupler fields object
+    coupler_fluxes = (;
+        :F_turb_ρτxz => CC.Fields.zeros(boundary_space),
+        :F_turb_ρτyz => CC.Fields.zeros(boundary_space),
+        :F_turb_energy => CC.Fields.zeros(boundary_space),
+        :F_turb_moisture => CC.Fields.zeros(boundary_space),
+        :temp1 => CC.Fields.zeros(boundary_space),
+        :temp2 => CC.Fields.zeros(boundary_space),
+    )
+
+    # Initialize the coupler fields so we can perform exchange
+    coupler_field_names = Interfacer.default_coupler_fields()
+    map(sim -> Interfacer.add_coupler_fields!(coupler_field_names, sim), values(model_sims))
+    coupler_fields = Interfacer.init_coupler_fields(FT, coupler_field_names, boundary_space)
+    flux_type = FluxCalculator.PartitionedStateFluxes()
+
+    # Step the atmosphere once to get non-zero wind and humidity
+    Interfacer.step!(atmos_sim, dt)
+
+    # Exchange the initial conditions between atmosphere and land
+    # This also tests the `get_field`, `update_field!` and `update_model_sims!` methods for `ClimaLandSimulation`
+    FieldExchanger.import_combined_surface_fields!(coupler_fields, model_sims, flux_type)
+    FieldExchanger.import_atmos_fields!(coupler_fields, model_sims, boundary_space, flux_type)
+    FieldExchanger.update_model_sims!(model_sims, coupler_fields, flux_type)
+
+    # Update land cache variables with the updated drivers in the cache after the exchange
```

Comment:
I think this is now updating land cache variables (including those needed for SF computation) only

Reply:
Yeah, that's what I tried to mean with this comment but maybe it isn't clear